make_prg/utils/seq_utils.py

```diff
@@ -234,3 +237,46 @@ def get_consensus_from_MSA(alignment: MSA) -> str:
             consensus_string_as_list.append(column.pop())
     consensus_string = "".join(consensus_string_as_list)
     return consensus_string
+
+
+def get_majority_consensus_from_MSA(alignment: MSA) -> str:
+    """
+    Produces a consensus string just based on the major base for each column.
+    Note: alignment needs to be upper-cased bases
```

Comment:
Why?

Reply:
This was not a good choice, fixed in https://github.com/iqbal-lab-org/make_prg/pull/60/commits/2089324c0e2bfa68ae21a002e449b81642c560e9